src/cryptoadvance/specter/templates/includes/js-helpers.jinja

```diff
@@ -0,0 +1,72 @@
+<!-- This file contains javascript helper functions that also can use jinja -->
+<script type="text/javascript">
+
+    // Formats an assetLabel (e.g. from a tx output) to a useful unit label
+    function formatAssetLabel(assetLabel, convertToSat){
```

Comment:
Not sure whether this is the best naming, perhaps rather _UnitLabel_ instead of _AssetLabel_? Asset is a bit misleading since it is mainly used for / in Liquid.

Reply:
`UnitLabel` is fine too :-)